vizro-core/docs/pages/user-guides/filters.md

```diff
@@ -62,7 +62,7 @@ automatically based on the target column, e.g. a dropdown for categorical data o
 
     [Filter]: ../../assets/user_guides/control/control1.png
 
-## Changing Selectors
+## Changing selectors
```

Comment:
Just applying it here, as I saw that you've implemented the same pattern in other parts of the docs. 

```suggestion
## How to change selectors
```